ThingIFSDK/ThingIFSDK/Command.swift

```diff
@@ -100,7 +86,7 @@ open class Command: NSObject, NSCoding {
      - Parameter alias: Alias to get action.
      - Returns Array of `AliasAction`.
      */
-    open func getAction(_ alias: String) -> [AliasAction] {
```

Comment:
@yongpingchen 
I got it. This method is wrong. I should implement method like following

```swift
public func getAction(_ alias: String, actionName: String) -> [AliasAction]
```

What do you think?